crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1115,36 +1115,10 @@ impl TryFrom<(&payments::BankRedirectData, Option<bool>)> for StripeBillingAddre
             }),
             payments::BankRedirectData::Ideal {
                 billing_details, ..
-            } => {
-                let billing_name = billing_details
-                    .clone()
-                    .and_then(|billing_data| billing_data.billing_name.clone());
-
-                let billing_email = billing_details
-                    .clone()
-                    .and_then(|billing_data| billing_data.email.clone());
-                match is_customer_initiated_mandate_payment {
-                    Some(true) => Ok(Self {
-                        name: Some(billing_name.ok_or(
-                            errors::ConnectorError::MissingRequiredField {
-                                field_name: "billing_name",
-                            },
-                        )?),
-
-                        email: Some(billing_email.ok_or(
-                            errors::ConnectorError::MissingRequiredField {
-                                field_name: "billing_email",
-                            },
-                        )?),
-                        ..Self::default()
-                    }),
-                    Some(false) | None => Ok(Self {
-                        name: billing_name,
-                        email: billing_email,
-                        ..Self::default()
-                    }),
-                }
-            }
+            } => Ok(get_stripe_sepa_dd_mandate_billing_details(
+                billing_details,
+                is_customer_initiated_mandate_payment,
+            )?),
```

Comment:
why are ideal changes done in sofort pr?

Reply:
Recurring pay for Ideal and Sofort in stripe happens through SEPA Direct Debit payments. Therefore both requires same request body resulting in repeated code, hence I removed the redundancy by moving it into a function get_stripe_sepa_dd_mandate_billing_details

Reply:
can u please add the same comment in code to avoid confusion in future